packages/extension-vscode/src/utils/analyze.ts

```diff
@@ -8,7 +8,7 @@ import { loadWebhint } from './webhint-packages';
 import { problemToDiagnostic } from './problems';
 import { promptRetry } from './prompts';
 
-const analyze = async (textDocument: TextDocument, webhint: import('hint').Analyzer): Promise<PublishDiagnosticsParams> => {
+const analyze = async (textDocument: TextDocument, webhint: import('hint').Analyzer): Promise<PublishDiagnosticsParams> => { // eslint-disable-line no-use-before-define
```

Comment:
I'm surprised eslint flagged this line for no-use-before-define. Any idea what was going on here or am I missing a reference that isn't defined until later in the file?

Reply:
I think it is an issue in `eslint` or `@typescript-eslint`. I tried to change `Analyzer` for another property and it was ok. TBH, I don't know why it is complaining.

Reply:
Got it. I could see why eslint might choke on the `import('hint').Analyzer` syntax. Hopefully this will get fixed later, but for now I'm fine with the one-off comment to disable.